README.md

```diff
@@ -1 +1,185 @@
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [https://github.com/ProgramKingAbel/Hello-Microverse/pull/1](https://google.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Code Editor preferrably Vs Code:
+
+```sh
+  https://visualstudio.microsoft.com
+```
+Github Account:
+```sh
+  https://github.com/
+```
+Npm installed: 
+```sh
+  https://nodejs.org/en/download
+```
+
+A LiveServer/ Localhost/ VS Code Live Server Extension
+
+A working and up to date Browser
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  git clone command
+```
+
+Install this project with:
+
+Linters:
+
+Lighthouse run:
+```sh
+  npm install -g @lhci/cli@0.7.x
+```
+Webhint run 
+```sh
+ npm install --save-dev hint@7.x
+```
+StyleLint run:
+```sh
+ npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+```
+
+### Usage
+
+To run the project, open the index.html file on your browser.
+
+
+### Run tests
+
+To run tests, run the following command:
+
+to check for styling errors:
+
+```sh
+  npx stylelint "**/*.{css,scss}"
+```
+To test optimization:
+```sh
+  lighthouse [site url]
+```
+
+### Deployment
+
+You can deploy this project using:
+
+Localhost on your computer through a local Server:
+
+
+```sh
+ https://www.apachefriends.org/download.html
+```
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Abel Morara**
+
+- GitHub: [@ProgramKingAbel](https://github.com/ProgramKingAbel)
+- Twitter: [@CEOAbel1](https://twitter.com/CEOAbel1)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/abelmatundamorara-451340250)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **[Make the application mobile responsive]**
+- [ ] **[Add media both images and videos]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project follow me on Github and give it a star to motivate build more cool staff. Thank you.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+I would like to thank the Microverse Community, My stand up team and coding partners who we work tirelessly to make it possible.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+
```

Comment:
- [ ] Your README file looks very professional 🤙 , however I noticed that the link attached to the MIT license is broken , increase the beauty of your README file by fixing this link with a MIT license.
- [ ] You could find it[ here](https://choosealicense.com/licenses/mit/)

Reply:
Working Environment set up with Linters - Stylelint, LightHouse and Webhint.
Developed homepage Markup and Styling.
Edit professional README.md file 
Installed NPM and JSON dependencies

Reply:
Hello @iambenkis Following the comment on your review, I have added a descriptive summary of my PR as well as added an MIT LICENSE.TXT file to the repository and fixed the broken MIT license broken link in my README.MD. Thank you for the guidance.